module/config.mjs

```diff
@@ -618,36 +617,39 @@ preLocalize("consumableTypes", { sort: true });
 
 /**
  * The valid currency denominations with localized labels, abbreviations, and conversions.
+ * The conversion number defines how many of that currency are equal to one GP (according
+ * to the rules as written).
  * @enum {{
```

Comment:
To make this more in line with other new configuration changes, this should probably be converted to a `typedef` which allows for a more detailed description of each property.

```suggestion
/**
 * Configuration data for a currency.
 *
 * @typedef {object} CurrencyConfiguration
 * @property {string} label         Localized label for the currency.
 * @property {string} abbreviation  Localized abbreviation for the currency.
 * @property {number} conversion    Number by which this currency should be multiplied to arrive at a standard value.
 */

/**
 * The valid currency denominations available in actors & containers.
 * @enum {CurrencyConfiguration}
```